apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/PluginInitializer.java

```diff
@@ -0,0 +1,50 @@
+package org.apache.skywalking.apm.agent.core.plugin;
```

Comment:
Error logs
> ERROR the following files don't have a valid license header: 
apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/PluginInitializer.java 

You need License header.